homework.md

```diff
@@ -92,13 +92,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Creating a save point for your files
```

Comment:
Committing changes is basically saving your file changes to your local repo

Reply:
Appreciate the feedback. Will be adding the changes soon thanks.